src/typings/react-native.d.ts

```diff
@@ -123,6 +123,9 @@ declare module 'react-native' {
         onLoadEnd?: (e: SyntheticEvent<Image>) => void;
         onLoadStart?: Function;
         onProgress?: Function;
+
+        //windows
```

Comment:
Minor: All comments should have a space after "//" and "Windows" should be capitalized for consistency.